addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -49,8 +49,9 @@ const LinkTools = Link.extend({
         this.$link.addClass('oe_edited_link');
         this.$button.addClass('active');
         return this._super(...arguments).then(() => {
-            if (!this.options.noFocusUrl) {
-                dom.scrollTo(this.$(':visible:last')[0], {duration: 0});
+            const lastOptionEl = this.$(':visible:last')[0];
+            if (!this.options.noFocusUrl && lastOptionEl) {
+                dom.scrollTo(lastOptionEl, {duration: 0});
```

Comment:
Should have a comment to explain how we can reach this code and not have a visible element, I am not sure to understand. Also, it should be for 14.4 I guess?

Reply:
I have not been able to reproduce it on 14.4. It seems to be related to the _onMouseDown method of the editor which differs between the two versions. I proposed another fix on this branch